docs/docs/index.md

```diff
@@ -3,23 +3,31 @@ authors: Fragcolor & contributors
 license: CC-BY-SA-4.0
 ---
 
-![](assets/HomePageLogo.png)
+![Fragnova Logo](assets/Fragnova-Logomark-01.png){: style="height:240px;width:300px"}
 
-# Home
+# Welcome to Fragnova documentation!
 
-Welcome to Fragnova documentation!
+Here at Fragnova, we're passionate about the potential of technology to change the way we interact with the world around us. 
 
-These documents will help you understand, use, and contribute to Fragnova technologies.
+We believe that we're on the brink of a major revolution, where traditional interfaces will be replaced by more flexible and intuitive virtual interfaces that can be seamlessly integrated into the user's environment.
 
-You can start from a high-level topic below or use the navigation links at the top.
+To make this vision a reality, we're dedicated to creating programming tools that will empower developers to build the future.
 
-- [Shards](./learn/shards/)
-- [Reference](./reference/)
-- [Contributions guides](./contribute/)
+<br>
 
-*Use the search box if you're looking for something specific.*
+<span style="font-weight:200;font-size:30px"> Ready to step into the world of Fragnova? </span>
 
+[Download](./download/){ .md-button .md-button--primary }
+
+<br>
+
+| | | |
+| :----------: | :----------: |:----------: |
+|[![](assets/GettingStartedLogo.png){: style="height:180px;width:180px"}](./learn/index.md) <br> <span style="font-weight:600;font-size:20px">[Getting Started](./learn/index.md)</span> | [![](assets/ShardsLogo.png){: style="height:180px;width:180px"}](./learn/shards/index.md)<br> <span style="font-weight:600;font-size:20px">[Shards](./learn/shards/index.md)</span> | [![](assets/RareEngineLogo_2.png){: style="height:180px;width:180px"}]()<br> <span style="font-weight:600;font-size:20px">[Rare Engine]()</span> |
+| | | |
```

Comment:
We should avoid having that much CSS in the middle of markdown. Image width is already a supported attribute. Font size shouldn't be changed in my opinion.

Otherwise there is no point to have a markdown engine and we can write the whole website in static HTML.

Reply:
The fonts were too small to support the appearance of a large and obvious navigational link.